docs/static/releasenotes.asciidoc

```diff
@@ -5,11 +5,25 @@ coming[8.0.0]
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-8-0-0-rc2,Logstash 8.0.0-rc2>>
 * <<logstash-8-0-0-rc1,Logstash 8.0.0-rc1>>
 * <<logstash-8-0-0-beta1,Logstash 8.0.0-beta1>>
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-0-0-rc2]]
+=== Logstash 8.0.0-rc2 Release Notes
+
+==== Notable issues fixed
+This release is mainly a consolidation of the code base.
```

Comment:
I don't think this sentence has much meaning for a user. I think we can take it out.